data.json

```
@@ -98,6 +98,15 @@
     "dateAdded": "2018-10-12T13:25:10.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "JS Bin",
+    "difficulty": "easy",
+    "description": "Fill out a Google form with your name and address and get a sticker. There’s no cost to you, but they will be restricting the send outs to bi-monthly or quaterly depending on demand.",
+    "reference": "https://jsbin.com/help/stickers/",
+    "image": "https://help.jsbin.com/images/stickers.jpg",
+    "dateAdded": "2018-01-31T15:43:00.000Z",
```

```suggestion
    "dateAdded": "2018-11-28T16:42:00.000Z",
```

dateAdded is based on issue creation date 🙂

---

Thanks for the suggestion. To be onest I struggled with defining this one.

---

No problem! Please let us know if you have any questions 😊